src/App.css

```diff
@@ -1,32 +1,258 @@
-.verticalLine {
-  border-right: thick solid #030000;
+@import url('https://kit.fontawesome.com/a076d05399.js');
+
+.Bookstore-CMS {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.user {
+  margin-left: 25%;
+  padding: 0.875rem;
+  border: 1px solid #e8e8e8;
+  border-radius: 50%;
+}
+
+.fa-user {
+  color: #0290ff;
+}
+
+.panel-bg {
```

Comment:
- [ ] Great job with the design :+1: Let's work on some more details to match the template.
  - There is a background with a different color around the design, kindly implement it. 😎️
  - The person icon on the navigation bar should be aligned according to the book cards. 🤝️
  - Please align the entire CMS in the center of the screen, it is currently aligned to the left and there is so much whitespace on the right side. 🥸️
  - Good job using the correct font family `Montserrat`, but I don't think it is taking any effect. I think you need to import it from Google fonts or someplace else. 🧐️
  - There is a separation between the navigation bar and the next section. They are using different colors, please implement them as per the design. 

**Zeplin design**
![Screenshot from 2023-03-11 12-44-42](https://user-images.githubusercontent.com/17243325/224477811-29e423fc-7061-45e2-9cfe-8a90fd29a6c5.png)


**Your design**
![Screenshot from 2023-03-11 12-43-17](https://user-images.githubusercontent.com/17243325/224477777-dc339df4-34fc-40e0-83ce-6e5730a373b4.png)

Reply:
thank you @amejid , i have implemented all your requirements.